doc/api/net.md

```diff
@@ -2,74 +2,133 @@
 
     Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an API for implementing network clients and servers.
+It can be accessed using:
+
+```js
+const net = require('net');
+```
 
 ## Class: net.Server
 <!-- YAML
 added: v0.1.90
 -->
 
-This class is used to create a TCP or local server.
+The `net.Server` class is used to implement TCP or local servers.
 
-`net.Server` is an [`EventEmitter`][] with the following events:
+All `net.Server` objects are [`EventEmitter`][] instances.
 
 ### Event: 'close'
 <!-- YAML
 added: v0.5.0
 -->
 
-Emitted when the server closes. Note that if connections exist, this
-event is not emitted until all connections are ended.
+The `'close'` event is emitted when the server closes. If active connections
+exist, this event is not emitted until all such connections are ended.
+
+The listener callback is called without passing any arguments.
+
+```js
+server.on('close', () => {
+  console.log('The server has been closed.');
+});
+```
 
 ### Event: 'connection'
 <!-- YAML
 added: v0.1.90
 -->
 
-* {net.Socket} The connection object
+The `'connection'` event is emitted when a new connection is established.
 
-Emitted when a new connection is made. `socket` is an instance of
-`net.Socket`.
+The listener callback is called with the `net.Socket` object for the connection
+passed as the only argument.
+
+```js
+server.on('connection', (socket) => {
+  console.log('A connection has been established');
```

Comment:
It would be a nice idea to maybe add a comment indicating usage for new readers, explaining what they can do to the socket. It might be obvious though.